src/codeCache.cpp

```diff
@@ -168,6 +168,20 @@ const void* CodeCache::findSymbolByPrefix(const char* prefix, int prefix_len) {
     return NULL;
 }
 
+const void* CodeCache::findSymbolByPrefix(const char* prefix, int prefix_len, char second_choice_if_contains) {
```

Comment:
Another function is redundant: this logic is relevant for all usages of `findSymbolByPrefix`, so I'd modify an existing function instead. Furthermore, `.` character does not need to be configurable, I don't forsee any other usages.